### PR TITLE
[instrumentation-fixes] make instrumentation occur

### PR DESCRIPTION
**Pull request by Valve**
only once per request and use naming convention suggested by Rails guides.
https://edgeguides.rubyonrails.org/active_support_instrumentation.html#creating-custom-events

> You should follow Rails conventions when defining your own events. The format is: event.library.

### CODE REVIEW

**Comment by Valve** [comment]
this PR is pretty simple, so I'll merge it. Please let me know if you see some issues and i"ll update it accordingly @drnluz

File: lib/solr/query/request/runner.rb [hunk 1]
```diff
@@ -22,10 +22,7 @@ def initialize(page:, page_size:, solr_params: {})
 
         def run
           raw_response = connection(PATH).post_as_json(request_params)
-          response = Solr::Response.from_raw_response(raw_response)
-          Solr.instrument(name: 'solrb.request_response_cycle',
-                          data: { request: request_params, response: raw_response })
-          response
```
Comment (drnluz) [inline lib/solr/query/request/runner.rb]: @Valve This instrumentation is needed for testing:
https://github.com/machinio/solrb/blob/master/lib/solr/testing.rb

Comment (Valve) [reply lib/solr/query/request/runner.rb]: @drnluz oh, I didn't realize it, should I return it?
Let's document this in the readme

Comment (drnluz) [reply lib/solr/query/request/runner.rb]: @Valve yes because without this we won't be able to run some specs on machinio

Comment (Valve) [reply lib/solr/query/request/runner.rb]: Got it, I'll return, please add this to readme when you get a chance

**Pull request merged**